lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -697,6 +697,11 @@ galaxy:
   # cleaned up in default Celery task configuration.
   #short_term_storage_cleanup_interval: 3600
 
+  # The file_source_temp_dir is a temporary directory used by file sources
```

Comment:
This looks manually generated. To update all config example files you should edit only `config_schema.yml` and run:
```
make config-rebuild
```

Reply:
Thank you! I have updated it now after running `make config-rebuild`